library/Ivoz/Provider/Domain/Model/Country/CountryDto.php

```diff
@@ -21,4 +23,32 @@ public static function getPropertyMap(string $context = '', string $role = null)
 
         return parent::getPropertyMap(...func_get_args());
     }
+
+    public function getNameEn(): string
+    {
+        $name = parent::getNameEn();
```

Comment:
why are you validating nullability in the dto?

Reply:
This reimplementation was added to make phpstan happy, but it's not loger required since https://github.com/irontec/ivoz-dev-tools/commit/4672770918e43d48f9a7a5662d974137cf820a9e